app/views/divisions/index_with_member.html.haml

```diff
@@ -9,5 +9,5 @@
 
 %p All votes #{@member.full_name_no_electorate} could have attended.
 
-- @member.person.members.order(entered_house: :desc).each do |member|
-  = render "divisions", member: @member, divisions: member.divisions_possible.order(date: :desc, clock_time: :desc, name: :asc)
+= render "divisions", member: @member, divisions: @member.divisions_possible(@date_start, @date_end).order(date: :desc, clock_time: :desc, name: :asc)
```

Comment:
This line is actually from my other PR #1127. The relevant line for this PR is the below one.